packages/klevu-ui/src/components/klevu-search-landing-page/klevu-search-landing-page.tsx

```diff
@@ -92,6 +100,11 @@ export class KlevuSearchLandingPage {
   async connectedCallback() {
     await KlevuInit.ready()
     await this.#fetchData()
+
+    const settings = getKMCSettings()
+    if (this.usePersonalisation === undefined && settings?.klevu_uc_userOptions.enablePersonalisationInSearch) {
+      this.usePersonalisation = true
```

Comment:
would it be better to have it as
this.usePersonlization = settings?.klevu_uc_userOptions.enablePersonalisationInSearch || false
so that it always is set with a boolean value true/false.